torchaudio/models/__init__.py

```diff
@@ -19,4 +20,5 @@
     'wav2vec2_base',
     'wav2vec2_large',
     'wav2vec2_large_lv60k',
+    '_Tacotron2',
```

Comment:
Why exporting the class suffixed with underscore?

Reply:
Previously I use underscore to represent prototype.
I've moved Tacotron2 to the `torchaudio.prototype` directory to avoid confusion.
Please let me know if this is the right place to put at this stage.
Thanks.